src/ServiceBusChannel.php

```diff
@@ -163,9 +145,32 @@ private function getUrl($endpoint): string
 
     public function generateTokenKey()
     {
+        if (empty($ventureConfig)) {
```

Comment:
this won't be needed if we do config initialisation on the `constructor()`

Reply:
@norbybaru I did not want to remove the test that using this so this is the best alternative.